plugins.json

```diff
@@ -647,6 +647,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.37"
     },
+    "spectre": {
+        "title": "Spectre Theme",
+        "version": "1.0.2",
+        "author": "Viggo",
+        "license": "CC-BY-4.0",
+        "description": "omizer plugin.",
+        "homepage": "Modern And Simple CSS Theme For Kanboard",
```

Comment:
typo in homepage and description fields?

Reply:
fixed

Reply:
> typo in homepage and description fields?

fixed